docs/installation/autoware/docker-installation.md

```diff
@@ -16,21 +16,21 @@
    git clone https://github.com/autowarefoundation/autoware.git
    ```
 
-2. Install the dependencies.
+2. You can install the dependencies either manually or using the provided Ansible script.
 
-   ```bash
-   ./setup-dev-env.sh docker
-   ```
+### Installing dependencies manually
 
-!!!warning
+- [Manual Installation Instructions Link](https://github.com/autowarefoundation/autoware/tree/main/ansible/roles/docker#manual-installation)
```

Comment:
`Install Docker and related tools` or something like that might be better to be consistent with the source installation page?
https://github.com/autowarefoundation/autoware-documentation/pull/63#pullrequestreview-929578917

Reply:
```suggestion
- [Install Docker and related tools](https://github.com/autowarefoundation/autoware/tree/main/ansible/roles/docker#manual-installation)
```
Sure